src/components/UI/Form/Input/Input.tsx

```diff
@@ -116,7 +116,9 @@ export const Input: React.SFC<InputProps> = ({ textArea = false, disabled = fals
           <FormHelperText className={styles.DangerText}>{form.errors[field.name]}</FormHelperText>
         ) : null}
         {helperText ? (
-          <FormHelperText className={styles.HelperText}>{helperText}</FormHelperText>
+          <FormHelperText id="helper-text" className={styles.HelperText}>
```

Comment:
Any reason why this id is added?

Reply:
Since nesting of component is too deep, and styling for specific case makes more difficult. 
We had a use case for the organization onboarding form, wherein helper text size was different and alignment was right aligned. So using conditional CSS and having id on the form field makes it easier to tackle such issues.